.github/workflows/run-lint.yaml

```diff
@@ -6,11 +6,15 @@ on:
       - 'packages/**'
       - 'index.yaml'
       - '**.md'
+    branches:
+      - master
   pull_request:
     paths-ignore:
       - 'packages/**'
       - 'index.yaml'
       - '**.md'
+    branches:
+      - master
```

Comment:
Why limiting this to PR and pushes to master?
Github Actions in public repos are free of charge, and it might be more convenient to see CI results before even opening a PR when working on something. Seems to me like it is fine to run it on all commits.

Reply:
previous configuration produced redundant pipelines (for branch and for PR), this change leaves pull requests pipelines, cause branches filter filters a base branch and disables branch pipelines for non-master branches.